lib/internal/test_runner/coverage.js

```diff
@@ -353,6 +354,7 @@ class TestCoverage {
         continue;
       }
       const { data, lineLengths } = sourceMapCache[url];
+      if (!data) throw new ERR_SOURCE_MAP_CANNOT_PARSE(url);
```

Comment:
```suggestion
      if (!data) throw new ERR_SOURCE_MAP_CORRUPT(url);
```

I think this change will answer @cjihrig concern, since the issue happens when using the map, not with parsing it